lib/utils/tsv_reader.rb

```diff
@@ -49,7 +49,7 @@ def records
     def line_to_hash(line)
       cols = line.split(@delim, -1)
       if cols.size != @header_hash.keys.size
-        raise IndexError, "Line cols: #{cols.size}, header cols: (#{@header_hash.keys.size})."
+        raise IndexError, "Line #{$.} has #{cols.size} cols, header has #{@header_hash.keys.size} cols."
```

Comment:
I'm surprised standardrb didn't complain about the perlism - we should prefer `require 'English'` and then we can say `$INPUT_LINE_NUMBER` instead of `$.`

Reply:
Thought I could get away with the occasional perlism but I will concede.